dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -197,27 +201,33 @@ public WorkflowScheme findScheme(String id) throws DotDataException {
 	}
 
 	@WrapInTransaction
-	public void saveSchemeForStruct(final Structure struc, final WorkflowScheme scheme) throws DotDataException {

```

Comment:
I am not sure if you should change the method signature, it could be better to create a new one and keep the old one, something like

```
@WrapInTransaction
 	public void saveSchemeForStruct(final Structure struc, final WorkflowScheme scheme) throws DotDataException {

       saveSchemeForStruct(struc, Arrays.asList(scheme));
```